.config/nixpkgs/flake.nix

```diff
@@ -35,20 +35,30 @@
       homeConfigurations = {
         "lucas@grease-lightning" = home-manager.lib.homeManagerConfiguration {
           pkgs = importPkgs "x86_64-linux";
-          modules = [ ./home.nix ];
+          modules = [ ./home ./home/linux ];
         };
+
+        # TODO: set up macbook:
+        # "lucas@my-macbook" = home-manager.lib.homeManagerConfiguration {
+        #   pkgs = importPkgs "aarch64-darwin";
+
+        #   modules = [ ./home ./home/darwin ];
+        # };
       };
       nixosConfigurations = {
         oolong = nixpkgs.lib.nixosSystem {
           system = "x86_64-linux";
           modules = [
             ./configuration.nix
+
             home-manager.nixosModules.home-manager
             {
               home-manager.useGlobalPkgs = true;
               home-manager.useUserPackages = true;
-              home-manager.users.lucas = import ./home.nix;
+              home-manager.users.lucas = import ./home;
```

Comment:
Note that I renamed my `/home.nix` file to `/home/linux.nix`.

My goal is to have separate home manager configs for my linux and macos machines.